Source/WarpX.cpp

```diff
@@ -181,6 +181,9 @@ bool WarpX::do_multi_J = false;
 int WarpX::do_multi_J_n_depositions;
 bool WarpX::safe_guard_cells = 0;
 
+std::map<std::string, amrex::MultiFab *> WarpX::multifab_map;
+std::map<std::string, amrex::iMultiFab *> WarpX::imultifab_map;
```

Comment:
Storing as raw pointers is not ideal for lifetime control.

We could either continue to store as unique pointers or store the object directly as `amrex::MultiFab`/`amrex::iMultiFab`, as we do in ImpactX.

cc @WeiqunZhang for guidance since I am at SC22.

Reply:
I agree that saving pointers can be problematic, but here, the lifetime of the MFs is controlled by the unique pointers in the WarpX class, e.g. `Efield_fp`. The map is only for convenience for the Python interface and isn't used anywhere in the C++.

Reply:
Yes, this should be a raw pointer because it does not own the memory.

Reply:
ah I overlooked that. makes sense (we could check if we like to migrate storage location later on)